frontend/cypress/integration/common/mesh.ts

```diff
@@ -53,6 +53,26 @@ When('user selects mesh node with label {string}', (label: string) => {
     });
 });
 
+When('user selects tracing mesh node', () => {
+  cy.waitForReact();
+  cy.get('#loading_kiali_spinner').should('not.exist');
+  cy.getReact('MeshPageComponent', { state: { isReady: true } })
+    .should('have.length', 1)
+    .then($graph => {
+      const { state } = $graph[0];
+
+      const controller = state.meshRefs.getController() as Visualization;
+      assert.isTrue(controller.hasGraph());
+
+      const { nodes } = elems(controller);
+      const node = nodes.find(n => n.getLabel() === 'jaeger' || n.getLabel() === 'Tempo');
+      assert.exists(node);
```

Comment:
`assertexpected undefined to exist` scenario fails here with OCP + sail setup. Will DM you details.

Reply:
Pushed a fix [I think]